client/components/atoms/ArrowInput/ArrowInput.js

```diff
@@ -0,0 +1,42 @@
+import React, { useEffect } from "react";
+import arrowIcon from "../../../assets/icons/icon_chevron-small-left.svg";
+import closeBtn from "../../../assets/icons/icon_close.svg";
+
+const ArrowInput = (props) => {
+  useEffect(() => {
+    return () => {};
+  }, [props.nameLeft, props.nameCenter, props.nameRight]);
+  return (
+    <div className="arrowInputWrap">
+      <div className="arrowInput">
+        <div className="arrowInputText_left">
+          {props.name_left}
+          <div className="arrowSign">
+            <img src={arrowIcon} />
+          </div>
+        </div>
+        <div className="arrowInputText_center">
+          {props.name_center}
```

Comment:
name_center or nameCenter?
nameCenter is the right convention by the way

Reply:
fixed:)